x-pack/plugins/alerting/server/rules_client/lib/get_alert_from_raw.ts

```diff
@@ -161,7 +171,12 @@ export function getPartialRuleFromRaw<Params extends RuleTypeParams>(
 
   if (omitGeneratedValues) {
     if (rule.actions) {
-      rule.actions = rule.actions.map((ruleAction) => omit(ruleAction, 'alertsFilter.query.dsl'));
+      rule.actions = rule.actions.map((ruleAction) => {
+        if (!isSystemAction(ruleAction)) {
```

Comment:
I think the omit, we will just have ignore it if this attribute did not exist in the object

Reply:
Yes, exactly. If it's a system action it does not have `alertsFilter` so we'll just skip `omit`.  And `if (!isSystemAction(ruleAction))` works as type-guard here as well.
Or what do you mean here?